src/backend/.docker/Dockerfile

```diff
@@ -6,6 +6,21 @@ WORKDIR /usr/src/app
 RUN apk update
 RUN apk add curl
 
+# Installs latest Chromium (100) package.
+RUN apk add --no-cache \
+      chromium \
+      nss \
+      freetype \
+      harfbuzz \
+      ca-certificates \
+      ttf-freefont \
+      nodejs \
+      yarn
```

Comment:
where are we making use of yarn?

Reply:
ahh my bad! Good catch!

Reply:
Infact, we don't even need nodejs as well